lib/features/mailbox/presentation/mixin/mailbox_widget_mixin.dart

```diff
@@ -60,6 +60,12 @@ mixin MailboxWidgetMixin {
         MailboxActions.markAsRead,
       MailboxActions.move,
       MailboxActions.rename,
+      if (mailbox.isSubaddressingAllowed)
+        MailboxActions.disallowSubaddressing
+      else
+        MailboxActions.allowSubaddressing,
+      if (mailbox.isSubaddressingAllowed)
+        MailboxActions.copySubaddress,
```

Comment:
we should check if capability `urn:apache:james:params:jmap:mail:shares` available or not. Please refer to the usage of `requireCapability` or `capability_validator.dart` for more information.

Reply:
done !